payroll-processor-functions/src/payroll-processor-functions/Features/Resources/ResourcesTrigger.cs

```diff
@@ -0,0 +1,30 @@
+using System;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Azure.WebJobs.Extensions.Http;
+using Microsoft.Extensions.Logging;
+
+namespace PayrollProcessor.Functions.Features.Resources
+{
+    public class ResourcesTrigger
+    {
+        [FunctionName("Resources_Create")]
+        public async Task<IActionResult> CreateEmployeesTable(
```

Comment:
I like to move this into another project so it can be executed independently of running the functions and so it isn't exposed to the client app.

Reply:
Yeah I would agree with this, as functionality increases we may see other more logical groupings of functions that could be broken out.